### PR TITLE
Feat/add preimplementation classes #33

### PR DESCRIPTION
**Pull request by aa-software2112**

### CODE REVIEW

File: app/helpers/date.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import datetime
```
Comment (lkadian) [inline app/helpers/date.py]: think we should scrap the helpers folder and only use utils folder. Both are synonymous I think

**Comment by samgo1** [comment]
@lkadian @KrishnaPatel1 Don't merge PR until im done including all my classes here

File: app/models/schedule.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    @abstractmethod
+    def get_schedule_date_string(self):
+        """
+
+        :return: Should return some date string representing
+         the schedule's representation scope (month or day) for front-end display
+        """
+        pass
+
+class DailySchedule(Schedule):
```
Comment (lkadian) [inline app/models/schedule.py]: Why not put these in their own file/module?
schedule.py
daily_schedule.py
monthly_schedule.py

Comment (samgo1) [reply app/models/schedule.py]: I don't mind having them in the same file

Comment (lkadian) [reply app/models/schedule.py]: I just remember in 343 we did the same thing and the file in the end was a huge mess, but if we feel like that won't happen we can keep it this way

Comment (aa-software2112) [reply app/models/schedule.py]: it's only for the three schedule classes, otherwise i'd agree w/ you

File: app/helpers/date.py (hunk 1 above)
Comment (aa-software2112) [reply app/helpers/date.py]: OK when we merge everything we should apply this, because now we have 4 branaches depending on the helper folder.

**Pull request merged**